### PR TITLE
Jormungandr: fix no reverse geocoding in experimental

### PR DESCRIPTION
**Pull request by TeXitoi**
http://jira.canaltp.fr/browse/ITI-216

### CODE REVIEW

File: source/jormungandr/jormungandr/georef.py [hunk 1]
```diff
@@ -48,8 +48,14 @@ def place(self, place):
         if place.startswith("coord:"):
             # In some cases, the given "from" is not always a findable address by kraken
             # we just forward the given coord and return a pt object
+            _, lon, lat = place.split(':')
```
Comment (antoine-de) [inline source/jormungandr/jormungandr/georef.py:51]: wouldn't it be better to remove the old `coord:` thing ?

Comment (TeXitoi) [reply source/jormungandr/jormungandr/georef.py:51]: That's a much bigger refactor.

Comment (pbougue) [reply source/jormungandr/jormungandr/georef.py:51]: Seems a bit big but might be nice

Comment (TeXitoi) [reply source/jormungandr/jormungandr/georef.py:51]: And it's opensource :innocent:

Comment (antoine-de) [reply source/jormungandr/jormungandr/georef.py:51]: rohhhh:

```
 ❯ git grep '"coord:' | wc -l
21
```

but I'm kidding, it's ok to do this later

**Pull request merged**